templates/shared_services/sonatype-nexus/terraform/deploy.sh

```diff
@@ -0,0 +1,14 @@
+# is this required?
```

Comment:
need to check

Reply:
I've been using these for running the TF locally using the make file. They should probably be consolidated into a single script used by all, outside of the terraform directories.

Reply:
Removed from here in the meantime